pkg/osquery/tables/airport/table_darwin.go

```diff
@@ -74,7 +74,7 @@ func generateAirportData(queryContext table.QueryContext, airportExecutor execut
 	options := tablehelpers.GetConstraints(queryContext, "option", tablehelpers.WithAllowedValues(allowedOptions))
 
 	if len(options) == 0 {
-		return nil, errors.Errorf("The %s table requires that you specify a constraint for option", tableName)
+		return nil, errors.Errorf("The %s table requires that you specify a constraint for WHERE option =. Valid values for option are (%s).", tableName, strings.Join(allowedOptions, ", "))
```

Comment:
I'd put this into a log line -- errors don't always make it back to the log.

Reply:
Correction, it does bubble up. So let's take it. 

```suggestion
		return nil, errors.Errorf("The %s table requires that you specify a constraint for WHERE option. Valid values for option are (%s).", tableName, strings.Join(allowedOptions, ", "))
```